From 14eebb0f972662df4ee94fe0a6b6b28dabdc9480 Mon Sep 17 00:00:00 2001
From: Josef Raschen <josef@raschen.org>
Date: Sat, 21 Mar 2026 15:39:24 +0000
Subject: [PATCH] Fix CMake minimum version

Upstream-Status: Pending
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 43a69ef..fdf088e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.12)
 PROJECT(mdio-tool)
 
 include(GNUInstallDirs)
-- 
2.43.0

